src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -265,7 +265,6 @@ def test_history_ram_cycle_information_representation():
     assert str(recreated_cycle_info) == str(cycle_info)
 
 
-@pytest.mark.skip(reason="TODO(sbethur): Enable running on simulated session. GitHub issue #1273")
```

Comment:
If this code is valid just cannot be run because of the other bug, consider leaving it in as skipped or commented out.

Reply:
Good idea. I left it in as skipped.